Util.Metaid.UnitTest/metaid.config

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
```

Comment:
XML vs json..  something to discuss maybe?

Reply:
Always XML for important things ! :-)  Could discuss making an xsd and generate serializing code, but the code for reading the file was there when i started :-)